examples/gears.c

```diff
@@ -322,6 +322,10 @@ int main(int argc, char *argv[])
         exit( EXIT_FAILURE );
     }
 
+    if (!glfwGetWindowAttrib(window, GLFW_TRANSPARENT)) {
```

Comment:
Style-wise, is it okay this places `{` on same line as `if`, while the rest of the code places it next line?

Reply:
Sorry that is an IDE mistake, I will fix that.